app/statistics/server/lib/statistics.js

```diff
@@ -114,7 +114,18 @@ export const statistics = {
 		statistics.totalLivechatAgents = Users.findAgents().count();
 
 		// livechat enabled
-		statistics.livechatEnabled = settings.get('Livechat_enabled');
+		statistics.livechatEnabled = settings.get('Livechat	_enabled');
+
+		// Count and types of omnichannel rooms
+		statistics.omnichannelSources = _.reduce(Promise.await(RoomsRaw.allRoomSourcesCount().toArray()), (acc, curr) => {
```

Comment:
why using lodash reduce instead of a normal reduce?

Reply:
I was just following the usage on the file, i can change to a regular reduce too.

Reply:
```suggestion
		statistics.omnichannelSources = Promise.await(RoomsRaw.allRoomSourcesCount().toArray()).map((source) => ({
			id: source._id.id,
			alias: source._id.alias,
			type: source._id.type,
			count: source.count,
		}));
```

Looking at the data, I found that the **map** operation fits better here instead of **reduce** operation since our original and final array always has a 1:1 mapping